coral/src/app/features/topics/request/form-schemas/topic-request-form.ts

```diff
@@ -47,7 +47,7 @@ const formSchema = z
     topicname: topicNameField,
     advancedConfiguration: advancedConfigurationField,
     remarks: z.string(),
-    description: z.string().min(1),
+    description: z.string().min(1).max(100),
```

Comment:
@programmiri possible to take a look at this minor FE change ?

In a topic request, we want to restrict the length of description, based on the size of column in db, which is 100.

Reply:
That works fine :) Validation is done correctly.